src/java/main/org/apache/jute/compiler/JType.java

```diff
@@ -27,7 +27,7 @@
 	private String mCName;
     private String mCppName;
     private String mCsharpName;
-    private String mJavaName;
+    protected String mJavaName;
```

Comment:
Thanks a lot @afine for pointing out the performance gains. I'd suggest to separate out the changes that helps to improve the performance from findbug fix because that would make the findbug fix/reviews simple. Also, iiuc perf related changes are applicable to all the branch codes and separate task would help us to track/merge the changes easily rather than clubbing multiple changes together in one commit.